principles/work_in_the_open.md

```diff
@@ -0,0 +1,49 @@
+# Work in the open
+
+> *Our code is open as early as possible. We only go private if we really, really, **really** have to!*
+
+[Point 8](https://www.gov.uk/service-manual/service-standard/make-all-new-source-code-open) of the [Digital Service Standard](https://www.gov.uk/service-manual/service-standard) states
+
+> You must make all new source code open to meet point 8 of the Digital Service Standard.
+
+So you could develop your code in private, and release in time for a [Beta](https://www.gov.uk/service-manual/agile-delivery/how-the-beta-phase-works) assessment, which (other factors aside) would be sufficient to pass.
+
+However we believe there are more benefits to making code public from the beginning, rather than starting private and releasing later.
+
+## Benefits
+
+### Free stuff
+
+Public repositories benefit from lots of free services. Continuous integration, dependency checking, code quality and test coverage are just a few of the things you can get for free if your project is open.
+
+### You've passed
+
+...point 8 of the Digital service standard (if you're a digital service going on [GOV.UK](https://gov.uk)). No last minute rush to release something just before an assessment, you're sorted from day one.
+
+### Flexibility
+
+It's easier to collaborate with colleagues when working on an open repo compared to a private one. Though you may have to grant access to allow them to make changes, there is nothing stopping them from looking at and referring to the code.
+
+### Visibility and sharing
+
+Projects that are open have more visibility which makes it easier for colleagues both In the Environment Agency and in other departments to see what we've built. This encourages collaboration rather than duplication.
```

Comment:
This is the first statement on the repo's root README

> These guides relate to the work of Digital Service teams in the Environment Agency who are building new Digital by default services.

I don't feel I have the powers to say this applies DEFRA wide, so wouldn't like to say, but I can ping something on yammer to try and improve visibility of it, as a first step to increasing its scope.

Coming back to your point, would it be better to reiterate that these principles apply to teams in the EA, on the `principles/` folder's root README?